src/js/stores/AppsStore.js

```diff
@@ -115,8 +115,15 @@ function getAppHealthWeight(health) {
   }, 0);
 }
 
+function calculateTotalResources(app) {
+  app.totalMem = app.mem * app.instances;
+  app.totalCpus = parseFloat((app.cpus * app.instances).toPrecision(4));
```

Comment:
@aldipower reminds me that actually we may want to process totalCpus further in the future, so it makes sense to have it as a float not a string.